pkg/cmd/repo/sync/git.go

```diff
@@ -70,12 +70,7 @@ func (g *gitExecuter) CurrentBranch() (string, error) {
 }
 
 func (g *gitExecuter) Fetch(remote, ref string) error {
-	args := []string{"fetch", "-q", remote, ref}
```

Comment:
This fetch used to be quiet but `GitClient.Fetch()` isn't quiet, I think

Reply:
Good catch. I changed it back to be quiet..